src/Tools/Source/RunTests/ProcessRunner.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace RunTests
+{
+    public sealed class ProcessOutput
+    {
+        private readonly int exitCode;
```

Comment:
Use readonly autoprops?  Also, this doesn't use the dotnet convention of _ for fields.